lib/jira/commands/install.rb

```diff
@@ -15,35 +15,52 @@ class Install < Base
 
       def run
         io.say('Please enter your JIRA information.')
-        inifile[:global] = params
+        inifile[:global] = base_params
+        inifile.write # Do this now because cookie authentication uses api calls
+
+        inifile.delete_section("cookie") if inifile.has_section?("cookie")
+        case authentication
+        when "basic"
+          inifile[:global][:password] = password
+        when "token"
+          inifile[:global][:token] = token
```

Comment:
There is token auth that allows a user to use a valid token after using some other means of getting one. Something to look at later is a simple integration with Sinatra for doing the session. That is out of scope for this change.